defaults/main.yml

```diff
@@ -82,7 +82,4 @@ OnAccessDisableDDD: "true"
 OnAccessExtraScanning: "true"
 OnAccessMountPath:
  - name: /home
- - name: /var
- - name: /var/lib/docker
```

Comment:
Scanning /var is what triggers recursively scanning tmp files (/var is a directory where tmp files are made) - either way it is scanned in the weekly root scan.